src/Models/Base/CdmChangeFeedContextBase.cs

```diff
@@ -0,0 +1,35 @@
+using Arcane.Framework.Services.Base;
+
+namespace Arcane.Stream.Cdm.Models.Base;
+
+public abstract class CdmChangeFeedContextBase : IStreamContext, IStreamContextWriter
```

Comment:
You can implement these interfaces directly, you don't need an abstract class here.